services/cad-service/nesting/engine.py

```
@@ -0,0 +1,166 @@
+            "rect",
+            attrib={
+                "x": "0",
+                "y": "0",
+                "width": str(sheet_width),
+                "height": str(sheet_height),
+                "fill": "#f9fafb",
+                "stroke": "#111827",
+                "stroke-width": "1",
+            },
+        )
+        for panel, placement in zip(panels, placements):
```

Ruff (0.14.1)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)